x/twap/types/keys.go

```diff
@@ -29,9 +29,16 @@ var (
 	historicalTWAPTimeIndexNoSeparator = "historical_time_index"
 	historicalTWAPPoolIndexNoSeparator = "historical_pool_index"
 
-	mostRecentTWAPsPrefix         = mostRecentTWAPsNoSeparator + KeySeparator
-	historicalTWAPTimeIndexPrefix = historicalTWAPTimeIndexNoSeparator + KeySeparator
-	historicalTWAPPoolIndexPrefix = historicalTWAPPoolIndexNoSeparator + KeySeparator
+	mostRecentTWAPsPrefix = mostRecentTWAPsNoSeparator + KeySeparator
```

Comment:
What's the reason we can't use the conventional way of iterating through all stores using prefix stores?

Reply:
What do you mean by the conventional way? Happy to change this if there is something more straight-forward